js/renovation/ui/toolbar/toolbar.tsx

```diff
@@ -8,18 +8,31 @@ import LegacyToolbar from '../../../ui/toolbar';
 
 import { DomComponentWrapper } from '../common/dom_component_wrapper';
 import { ToolbarProps } from './toolbar_props';
+import { isDefined } from '../../../core/utils/type';
 
 export const viewFunction = ({
   props,
   restAttributes,
-}: Toolbar): JSX.Element => (
-  <DomComponentWrapper
-    componentType={LegacyToolbar}
-    componentProps={props}
-  // eslint-disable-next-line react/jsx-props-no-spreading
-    {...restAttributes}
-  />
-);
+}: Toolbar): JSX.Element => {
+  const { rtlEnabled, items } = props;
+
+  items?.forEach((item) => {
+    if (typeof item !== 'string') {
+      if (!isDefined(item.options)) {
+        item.options = {};
+      }
+      item.options.rtlEnabled = rtlEnabled ?? false;
```

Comment:
We shouldn't rewrite item.options.rtlEnabled if it's defined

Reply:
And may be we should get some portion of logic regarding getting rtlEnable from context

Reply:
Because props.rtlEnabled it's only one way to get real value of rtlEnabled

Reply:
Yep, my mistake.
Fixed